spec/helpers/lib/file-creator.js

```diff
@@ -0,0 +1,42 @@
+"use strict";
+
+module.exports = (cryptoAsync, fsAsync) => {
+    /**
+     * Appends 1MB of data to the file provided. I only do this so that
+     * I don't have to load the entire contents of the file into memory.
```

Comment:
We talked about this, and I realized that there isn't really a way to use streams with `crypto.randomBytes()`.